### ISSUE TITLE
Make numbers the same height.

### ISSUE THREAD

**Issue by tylernathanreed** [impallari/Raleway#19]
![image](https://cloud.githubusercontent.com/assets/6486381/20896708/a994b638-bae4-11e6-82a9-d6b725a7da28.png)

Whenever I'm looking at a string of numbers, the fact that they don't line up always irks me. If you try to line them up with letters, it looks terrible:

![image](https://cloud.githubusercontent.com/assets/6486381/20896659/676e2afa-bae4-11e6-844d-6462b78dc87d.png)

It's like the numbers use their own alignment, which makes certain number look bad.

Personally, I think that they should line up with the Capital Letters (Like 6 and 8).

1, 2, and 0 may need to be resized. If I align the numbers by their bottoms, it comes out looking like this:

![image](https://cloud.githubusercontent.com/assets/6486381/20896891/73dcc098-bae5-11e6-8dd4-377ec96d7131.png)

The tops of 1, 2, and 0 are mostly aligned with each other, but they're not aligned with any other character.

Seeing stuff like this makes me think that the font sizes are different:

![image](https://cloud.githubusercontent.com/assets/6486381/20896963/b9c47a38-bae5-11e6-8e13-37c23eed1871.png)

Whatever alignment you guys decide to use, the numbers themselves should at least align with each other. The numbers lack consistency, and I'd love to see that issue be fixed.

**Comment by impallari** [comment]
Default are "old style figures". "Lining figures" are also included in the font. they are accessible via Open Type features.
Do some google search on "old style figures" and "linining figures". You will learn something new, and discover that many fonts have both :)

**Comment by davelab6** [comment]
Pablo, I hear about this really a crazy amount of the time. I would like to
propose making the lining figures default. Would you be willing to do this
to make users happy? 😄

**Comment by dom1n1k** [comment]
> propose making the lining figures default.

I agree. "Old style" figures - it's fine and useful, but usually "lining" more suitable.
Besides, some web-converters remove opentype options by default.

**Comment by Anyasia** [comment]
If some web converters remove opentype options by default, I propose that's even more reason to maintain the old-style figures. They're really one of the main reasons I use the font.

**Comment by joarobles** [comment]
Hi there! @impallari is making the lining figures default going to be considered?
